Python/Product/Profiling/Profiling/LaunchProfiling.xaml

```diff
@@ -152,6 +152,14 @@
                 </Grid>
             </GroupBox>
         </Grid>
+
+        <Grid Grid.Row="1">
+            <CheckBox Grid.Row="1" Margin="12 12 12 4"
+                      Content="{x:Static ui:Strings.LaunchProfiling_MixedModeProfiling}"
+                      IsEnabled="{Binding IsVTuneAvailable}"
```

Comment:
`IsVTuneAvailable` may not be compiled in. I think you should keep it always compiled in.

We should set the `Visibility` attribute so that if support is not compiled in, the control is Hidden+Collapsed.

There's examples elsewhere in PTVS sources that show how to set Visibility binding to a bool property.

Reply:
Fixed in a57dc046b , perhaps a little artlessly.  This is because there's a sequence of decisions more than a single boolean (whether external profiler driver is compiled, if so, whether vtune is installed, and if so, whether the user wants to make use of it)